website/docs/basics/the-structure.md

```diff
@@ -0,0 +1,185 @@
+---
+id: the-structure
+title: The Structure
+sidebar_label: The Structure
+---
+
+[![docs-source](https://img.shields.io/badge/source-eigthshift--libs-blue?style=for-the-badge&logo=php&labelColor=2a2a2a)](https://github.com/infinum/eightshift-libs)
+
+If you followed the chapters this far and set your first project with all classes from the `wp boilerplate init_theme` command, you saw that you have an `src` folder in the root of your project.
+
+In that root folder, you have a bunch of folders with classes. We like to structure our projects by features, so for example, all `custom post type` registrations will be located in the `CustomPostType` folder.
+
+Using the [WP-CLI](wp-cli) commands, you follow our folder structure so that all of your projects will have the same layout. This helps us when a new developer comes to the project, and he or she can know where things are from the start.
```

Comment:
```suggestion
Using the [WP-CLI](wp-cli) commands, you follow our folder structure so that all of your projects will have the same layout. This helps us when a new developer comes to the project, and they can know where things are from the start.
```